tools/update-prod-static

```diff
@@ -56,16 +57,18 @@ run(['./tools/minify-js'] + (['--prev-deploy', prev_deploy] if prev_deploy else
     stdout=fp, stderr=fp)
 
 # Copy the KaTeX files outside node_modules
-run(['mkdir', '-p', os.path.join(settings.STATIC_ROOT, 'node_modules/katex/dist/')],
-    stdout=fp, stderr=fp)
+os.makedirs(os.path.join(settings.STATIC_ROOT, 'node_modules/katex/dist/'),
+            exist_ok=True)
 
-run(['cp', 'node_modules/katex/dist/katex.css',
-     os.path.join(settings.STATIC_ROOT, 'node_modules/katex/dist/')],
-    stdout=fp, stderr=fp)
+shutil.copy('node_modules/katex/dist/katex.css',
+            os.path.join(settings.STATIC_ROOT, 'node_modules/katex/dist/'))
 
-run(['cp', '-R', 'node_modules/katex/dist/fonts',
-     os.path.join(settings.STATIC_ROOT, 'node_modules/katex/dist/fonts')],
-    stdout=fp, stderr=fp)
+for dirpath, dirnames, filenames in os.walk('node_modules/katex/dist/fonts'):
+    os.makedirs(os.path.join(settings.STATIC_ROOT, dirpath), exist_ok=True)
+    for filename in filenames:
+        shutil.copy(
+            os.path.join(dirpath, filename),
+            os.path.join(settings.STATIC_ROOT, dirpath, filename))
```

Comment:
I think I liked this block better as a `cp -R`; it's more readable, and I think that's more important for something like this.